dijet/tasks/jer.py

```diff
@@ -60,23 +60,28 @@ def output(self) -> dict[law.FileSystemTarget]:
         return outp
 
     def run(self):
-        widths = self.load_alpha()
+        widths = self.load_widths()
 
         # ### Now JER SM Data
+        jer = widths["fits"].copy()
+        view = jer.view()
```

Comment:
This is not the JER yet here, but the width of the distributions. I understand this is only a copy that will be overwritten with the actual JERs later, but this makes it difficult understand what is being read in by the task. I would also not create a view here, but rather at the very end before filling the JERs into an output histogram.

Adopting the naming scheme suggested above, this would need to be changed to:
```suggestion
        widths = results_extrapolation["intercepts"].copy()
```